core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7606DependencyImportScopeTest.java

```diff
@@ -0,0 +1,46 @@
+package org.apache.maven.it;
+
+import java.io.File;
+
+import org.apache.maven.shared.verifier.Verifier;
+import org.apache.maven.shared.verifier.util.ResourceExtractor;
+import org.junit.jupiter.api.Test;
+
+/**
+ * This is a test set for <a href="https://issues.apache.org/jira/browse/MNG-7606">MNG-7606</a>.
+ * It checks that "import" scope for dependencies work
+ *
+ */
+class MavenITmng7606DependencyImportScopeTest
+    extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng7606DependencyImportScopeTest()
+    {
+        super( "[3.9.0,)" );
```

Comment:
What would be an appropriate lower bound here?

Reply:
This should work with all Mavens, no?

Reply:
Agreed, adjusted accordingly in https://github.com/apache/maven-integration-testing/pull/214/commits/e192fd6384aaeb266ce14b2a95a24947887dd3d9